src/components/TitleBar.tsx

```diff
@@ -40,14 +43,22 @@ const TitleBar: React.FC<TitleBarProps> = ({
       className={`h-titlebar  flex justify-between`}
       style={{ backgroundColor: "#303030" }}
     >
+
       <div
-        className=" flex"
+        className="flex"
         style={
           platform === "darwin"
             ? { marginLeft: "70px" }
             : { marginLeft: "10px" }
```

Comment:
```suggestion
            : { marginLeft: "2px" }
```

Reply:
<img width="67" alt="Screenshot 2024-03-30 at 21 42 20" src="https://github.com/reorproject/reor/assets/17236551/700c3d04-3b70-492e-b6d4-d06839f9cb6a">
Small suggestion to make the left button be centered with the sidebar :)